app/src/main/java/de/danoeh/antennapod/config/StorageCallbacksImpl.java

```diff
@@ -125,6 +125,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     PodDBAdapter.KEY_CHAPTER_TYPE));
         }
         if(oldVersion <= 14) {
```

Comment:
The current release version should still be db version 14.
I'm not sure if it is a good idea to have several db upgrades within one release cycle, may become harder to test that the db upgrades correctly.